src/systems/systems/src/hydraulic/brake_circuit.rs

```diff
@@ -496,6 +583,75 @@ impl Default for AutobrakeDecelerationGovernor {
     }
 }
 
+pub struct BrakeAccumulatorCharacteristics {
+    total_volume: Volume,
+    gas_precharge: Pressure,
+    target_pressure: Pressure,
+    volume_at_init: Volume,
+}
+impl BrakeAccumulatorCharacteristics {
+    // 1/20 would mean standard deviation is "full accumulator volume / 20"
+    const STANDARD_DEVIATION_RATIO_FROM_FULL_INIT_VOLUME: f64 = 1. / 20.;
+
+    // Real accumulator is considered ok if +/- 50psi from charts
+    // 16 psi std dev would give 99.7% of values inside those 50 expected psi according to normal distribution
+    const STANDARD_DEVIATION_FOR_GAS_PRE_CHARGE_DISTRIBUTION_PSI: f64 = 16.;
+
+    pub fn new(
+        total_volume: Volume,
+        gas_precharge: Pressure,
+        target_pressure: Pressure,
+        empty_after_maintenance_probability: Ratio,
+    ) -> Self {
+        let is_empty =
+            random_from_range(0., 1.) < empty_after_maintenance_probability.get::<ratio>();
+
+        let actual_gas_precharge_randomized =
+            Pressure::new::<psi>(random_from_normal_distribution(
+                gas_precharge.get::<psi>(),
+                Self::STANDARD_DEVIATION_FOR_GAS_PRE_CHARGE_DISTRIBUTION_PSI,
+            ));
+
+        let init_volume_for_target_pressure =
+            total_volume - (actual_gas_precharge_randomized * total_volume) / target_pressure;
```

Comment:
If you change the order of the multiplication and division you get higher precision with the same performance.